code/datums/ai/human.dm

```diff
@@ -0,0 +1,47 @@
+/datum/ai/human
+	name = "human"
+
+/datum/ai/human/do_process(var/time_elapsed)
+	var/mob/living/carbon/human/H = body
+	if(H.stat != CONSCIOUS)
+		return
+
+	if(H.get_shock() && H.shock_stage < 40 && prob(3))
+		H.emote(pick("moan","groan"))
+
+	if(H.shock_stage > 10 && prob(3))
+		H.emote(pick("cry","whimper"))
+
+	if(H.shock_stage >= 40 && prob(3))
+		H.emote("scream")
+
+	if(!H.restrained() && H.lying && H.shock_stage >= 60 && prob(3))
+		H.custom_emote("thrashes in agony")
+
+	if(!H.restrained() && H.shock_stage < 40 && prob(3))
+		var/maxdam = 0
+		var/obj/item/organ/external/damaged_organ = null
+		for(var/obj/item/organ/external/E in H.organs)
+			if(!E.can_feel_pain()) continue
+			var/dam = E.get_damage()
+			// make the choice of the organ depend on damage,
+			// but also sometimes use one of the less damaged ones
+			if(dam > maxdam && (maxdam == 0 || prob(50)) )
+				damaged_organ = E
+				maxdam = dam
```

Comment:
Probably should `break`.

Reply:
I don't want to change this logic for fear of regression.